tests/embedding/test_qk_eng_wb2.py

```diff
@@ -291,10 +292,81 @@ def test_image_export(printer, selection, embedded_app):
     )
     settings_720p.Background = Ansys.Mechanical.DataModel.Enums.GraphicsBackgroundType.White
     settings_720p.Width = 1280
-    # settings_720p.Capture = Ansys.Mechanical.DataModel.Enums.GraphicsCaptureType.ImageOnly
     settings_720p.Height = 720
     settings_720p.CurrentGraphicsDisplay = False
     ExtAPI.Graphics.ExportImage(
         os.path.join(os.getcwd(), "geometry.png"), image_export_format, settings_720p
     )
     assert os.path.isfile(os.path.join(os.getcwd(), "geometry.png"))
+
+
+@pytest.mark.embedding
```

Comment:
put this in another file, not `test_qk_eng_wb2.py`.  This file is intended to contain the port of some QA tests to pymechanical.